packages/components/paginator/src/items-counter.ts

```diff
@@ -0,0 +1,95 @@
+import { localized, msg, str } from '@lit/localize';
+import { type CSSResultGroup, LitElement, type PropertyValues, type TemplateResult, html } from 'lit';
+import { property, state } from 'lit/decorators.js';
+import styles from './items-counter.scss.js';
+
+declare global {
+  interface HTMLElementTagNameMap {
+    'sl-items-counter': ItemsCounter;
+  }
+}
+
+/**
+ * A component that can be used with the paginator component.
+ * Contains information about currently visible items on the page and total amount of items.
+ */
+@localized()
+export class ItemsCounter extends LitElement {
+  /** @internal */
+  static override styles: CSSResultGroup = styles;
+
+  /** Currently active page, if not set - default to 1. */
+  @property({ type: Number, attribute: 'active-page' }) activePage = 1;
+
+  /** @internal Currently visible items on the current page. */
+  @state() currentlyVisibleItems = 1;
+
+  /** Items per page, if not set - default to 10. */
+  @property({ type: Number, attribute: 'items-per-page' }) itemsPerPage = 10;
+
+  /** @internal Pages amount. */
+  #pages = 1;
```

Comment:
Please put private variables, *after* static variables and before non-private vars :)